src/components/dialogs/export-case-dialog.tsx

```diff
@@ -109,6 +123,20 @@ const ExportCaseDialog = (props: ExportCaseDialogProps) => {
                 {intl.formatMessage({ id: 'download.export.button' })}
             </DialogTitle>
             <DialogContent>
+                {oneFileMode && (
+                    <TextField
+                        key="fileName"
+                        margin="dense"
+                        label={<FormattedMessage id="download.fileName" />}
+                        variant="filled"
+                        id="fileName"
+                        value={fileName}
+                        style={{ width: '100%' }}
+                        onChange={(event: any) =>
```

Comment:
to remove type any

Reply:
As you wish. But what difference does it make ?